src/Main.hs

```diff
@@ -16,7 +16,7 @@ import Data.Monoid
 import Data.Default
 import Data.ByteString.Lazy.Char8 (unpack)
 
-import Options.Applicative hiding (maybeReader)
```

Comment:
This is specially to be compatible with optparse-applicative-0.12 since Exherbo only have `0.12.1.0` at the moment.
So this change should wait until we'll get `0.13` in `::haskell`.

Note that it produce warning only build it with `0.12`. It will not produce any warnings with `0.13`.